src/task.c

```diff
@@ -215,6 +278,15 @@ task_create_thread_with_hooks(platform_thread       *thread,
    return ret;
 }
 
+/*
+ * -----------------------------------------------------------------------------
+ * task_thread_create() - Helper method to create a new task and to do the
+ *  required registration with Splinter.
+ *
+ * Currently, this is active mainly in tests. It's also used to create
+ * background tasks, a feature that is currently not enabled.
```

Comment:
@ajhconway - RESOLVE: What's the history on background threads?

Can I elicit some documentation from you and put it down here, so future readers can benefit?

Reply:
Thanks, @ajhconway - I will update the prolog accordingly.

Have opened issue #450 requesting that we have to work on re-enabling background threads support.